Packs/ProofpointServerProtection/Integrations/ProofpointProtectionServerV2/ProofpointProtectionServerV2.yml

```diff
@@ -0,0 +1,548 @@
+      isArray: false
+      name: brand_template
+      required: false
+      secret: false
+    - default: false
+      description: The Secure Reader response profile to use when a release is used for an encrypted message.The Response Profiles are listed on the Information Protection > Encryption > Response Profiles page in the management interface (admin UI).
+      isArray: false
+      name: security_policy
+      required: false
+      secret: false
+    deprecated: false
+    description: Releases the message to the email infrastructure without further scanning. The message remains in the folder and will be moved to the `deleted_folder` if specified.
+    execution: false
+    name: proofpoint-pps-release-message
+  - arguments:
+    - default: false
+      description: Name of the folder where the message is stored (e.g., HIPAA).
+      isArray: false
+      name: folder_name
+      required: true
+      secret: false
+    - default: false
+      description: |-
+        Comma-separated list of message GUIDs. Format is folder_id:table_id:dbase_id (e.g., 4:2:6), or in Cloud Quarantine format is GUID (e.g., g4fsnj_sTLMk9hECaJ wmmxwP6lQkr5k7).
+      isArray: false
+      name: local_guid
+      required: true
+      secret: false
+    deprecated: false
+    description: Resubmits the message to the filtering modules. The message is removed from the folder and will not be moved to any folder.
+    execution: false
+    name: proofpoint-pps-resubmit-message
+  - arguments:
+    - default: false
+      description: Name of the folder where the message is stored (e.g., HIPAA).
+      isArray: false
+      name: folder_name
+      required: true
+      secret: false
+    - default: false
+      description: |-
+         Comma-separated list of message GUIDs. Format is folder_id:table_id:dbase_id (e.g., 4:2:6), or in Cloud Quarantine format is GUID (e.g., g4fsnj_sTLMk9hECaJ wmmxwP6lQkr5k7).
+      isArray: false
+      name: local_guid
+      required: true
+      secret: false
+    - default: false
+      description: Name of the folder to move the message to. The folder must be for quarantined messages from the same type of module. For example, you cannot send deleted spam messages to a folder for deleted DLP Incidents, and vice versa.
+      isArray: false
+      name: deleted_folder
+      required: false
+      secret: false
+    - default: false
+      description: The new subject with which to overwrite the original subject for the message.
+      isArray: false
+      name: subject
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'false'
+      description: 'Whether to append the original subject to the string specified in the subject argument. Possible values are: "true" and "false". Default is "false".'
```

Comment:
no need to specify defaults, the server adds them automatically

Reply:
@ShirleyDenkberg FYI